### ISSUE TITLE
How to start to debug with frontend application

### ISSUE THREAD

**Issue by NiceLisan** [colinin/abp-vue-admin-element-typescript#44]
Hi, can you write documents on how to start the frontend application with more-detail?
for example: 
1:the suitable vue-cli version?
2:the yarn install dependancy? which is the suitable directory to run it?
for me, i got the following error and yarn failed

```
E:\abp-vue-admin-element-typescript\vueJs>yarn install
yarn install v1.19.1
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
warning @microsoft/signalr > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning @microsoft/signalr > request > har-validator@5.1.5: this
warning element-ui > async-validator > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning jsonlint > nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
error An unexpected error occurred: "https://github.com/seonim-ryu/Squire: connect ETIMEDOUT 192.30.253.112:443".
info If you think this is a bug, please open a bug report with the information provided in "E:\\abp-vue-admin-element-typescript\\vueJs\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

3:how to distinguish the vuejs and vuejsempty?

in fact , i couldnot run the following cmd success, only in vuejsempty folder  i can yarn install , but i couldnot start the vue project to see the front-end . where should i input the right configuration.
```
cd vueJs

start-vue-admin.bat
```

4: how to fill the follow default address according to the services in the source project:
```

VUE_APP_BASE_API = '你的api网关服务器地址'

VUE_APP_BASE_IDENTITY_SERVICE = '你的identityService服务器地址'

VUE_APP_BASE_IDENTITY_SERVER = '你的identityServer服务器地址'
```

**Comment by colinin** [comment]
Please read the README document carefully

Front-end project description：[vue-typescript-admin-template](https://github.com/Armour/vue-typescript-admin-template)

**Comment by NiceLisan** [comment]
sorry , i've read the doc so many times  but still run failed; 
i suppose you 'd better write with more detail in readme document

> Please read the README document carefully
> 
> Front-end project description：[vue-typescript-admin-template](https://github.com/Armour/vue-typescript-admin-template)

**Comment by NiceLisan** [comment]
> sorry , i've read the doc so many times but still run failed;
> i suppose you 'd better write with more detail in readme document
> 
> > Please read the README document carefully
> > Front-end project description：[vue-typescript-admin-template](https://github.com/Armour/vue-typescript-admin-template)

i can easily run the Armour's vue template project ; but can't run the abp-vue frontend projects.

this is  my problem.

**Comment by colinin** [comment]
Did you follow this procedure?

```
Project setup
With yarn or npm

Install dependencies
yarn install
Custom vue project config
.env.Github.production rename to .env.production
vue.config.github.js rename to vue.config.js

Modify the server address that the development environment will use for the proxy. Provide the following three addresses: IdentityService, IdentityServer, and ApiService

    proxy: {
      [process.env.VUE_APP_BASE_IDENTITY_SERVER]: {
        target: 'your identityService address',
        changeOrigin: true,
        pathRewrite: {
          ['^' + process.env.VUE_APP_BASE_IDENTITY_SERVER]: ''
        }
      },
      [process.env.VUE_APP_BASE_IDENTITY_SERVICE]: {
        target: 'your identityServer address',
        changeOrigin: true,
        pathRewrite: {
          ['^' + process.env.VUE_APP_BASE_IDENTITY_SERVICE]: ''
        }
      },
      [process.env.VUE_APP_BASE_API]: {
        target: 'you api gateway address',
        changeOrigin: true,
        pathRewrite: {
          ['^' + process.env.VUE_APP_BASE_API]: ''
        }
      }
    }
Modify the actual address of the production environment, as above

# Base api
# Remeber to change this to your production server address
# Here I used my mock server for this project
# VUE_APP_BASE_API = 'https://vue-typescript-admin-mock-server.armour.now.sh/mock-api/v1/'

VUE_APP_BASE_API = 'your api gateway address' // Just fill in the api gateway address

VUE_APP_BASE_IDENTITY_SERVICE = 'your identityService address' //Corresponding background Identity service address, generally gateway address

VUE_APP_BASE_IDENTITY_SERVER = 'your identityServer address' // Corresponding background IdentityServer4,

# client id
VUE_APP_CLIENT_ID = 'vue-admin-element'
# client secret
VUE_APP_CLIENT_SECRET = '1q2w3e*'


Compiles and hot-reloads for development
cd vueJs

start-vue-admin.bat

```

**Comment by NiceLisan** [comment]
@GitXieJun
Now, i've managed to run all the backend services success, Can you help me how to start the front-end project, sincere thanks~

**Comment by colinin** [comment]
step1、**cd vueJs**

step2、**npm install**

step3、**start-vue-admin.bat** or **npm run serve-nomock**

**Comment by NiceLisan** [comment]
> step1、**cd vueJs**
> 
> step2、**npm install**
> 
> step3、**start-vue-admin.bat** or **npm run serve-nomock**

ofcourse, i've tried to follow the instruction many times but got the failed result.

#53 https://github.com/colinin/abp-vue-admin-element-typescript/issues/53
maybe there are some missing codes and files in  the source codes you have uploaded ?
it's funny that many people reported the failed issue of  front-end codes .  

sincerely hope that you can clone the source code in the github by yourselves to see if you can reprouduce the same issues. 

many thanks.

**Comment by NiceLisan** [comment]
finally , i found you have missed to upload the below folder and related files:

**views/task/index.vue**
```


Failed to compile.

./src/router/modules/task.ts
Module not found: Error: Can't resolve '@/views/task/index.vue' in 'E:\FM_Tech\abp-vue-admin-element-typescript\vueJs\src\router\modules'
```

**Comment by colinin** [comment]
Ok, I'll get rid of this useless view the next time I merge

**Comment by NiceLisan** [comment]
> Ok, I'll get rid of this useless view the next time I merge

it's ok, so it's appreciated that you close this issue the next time you merge and push the source code. 

thanks.

**Comment by colinin** [comment]
y colleagues and I cloned it several times. Build: Prod is ok, maybe it's your local environment problem

**Comment by colinin** [comment]
The Task view has been removed
